docs/source/install/profile.rst

```diff
@@ -38,6 +44,12 @@ Queue: gpu (4x NVIDIA P100 16GB)
 Hypnos (HZDR)
 -------------
 
+**System overview:** `link (internal) <https://www.hzdr.de/db/Cms?pOid=29813>`_
+
+**User guide:** *None*
```

Comment:
Isn't there a funny web server mini-wiki running on `http://hypnos4/wiki` or so?

Reply:
I do not know - that is the first time I hear this.

Reply:
it is `http://hypnos3/wiki` but only available within our intranet.